src/Action.hs

```diff
@@ -382,13 +395,15 @@ validLoggedIn us = isJust (us ^. usUserId) && isJust (us ^. usSessionToken)
 validUserState :: UserState -> Bool
 validUserState us = us == userLoggedOut || validLoggedIn us
 
-getSpacesForCurrentUser :: (ActionUserHandler m, ActionPersist m) => m [IdeaSpace]
+getSpacesForCurrentUser :: (ActionUserHandler m, ActionPersist m, ActionLog m) => m [IdeaSpace]
 getSpacesForCurrentUser = do
     user <- currentUser
     query $ getSpacesForRoles (user ^. userRoleSet)
 
-deleteUser :: (ActionPersist m) => AUID User -> m ()
-deleteUser = update . DeactivateUser
+deleteUser :: (ActionSessionLog m, ActionPersist m) => AUID User -> m ()
+deleteUser uid = do
+    logEvent INFO $ cshow uid <> " is deactivated"
```

Comment:
that sounds like the message i would expect if the user has been deactivated befre, and this action can't do it any more and is warning about it.  `deactivating user` is less confusing.  i'll change this.